lib/core/src/lib/dialogs/unsaved-changes-dialog/unsaved-changes-dialog.component.scss

```diff
@@ -18,11 +18,12 @@ adf-unsaved-changes-dialog {
 
         &-discard-changes-button {
```

Comment:
can we avoid using custom colors and just use the material one? i.e. `button color="warn"` in that case we inherit the global theming

Reply:
So two things here: 
1. For my work I don't need this color for button. I need it just for red border for message, below screenshot from design: 
<img width="774" alt="image" src="https://github.com/user-attachments/assets/f0497ceb-a764-4bc2-a777-4eb60fb5ed70">

I noticed we have already this color for this button in that file so the only thing which I did was renaming variable used by that button to can use this also for my place. So my work has nothing with button. 

2. We can't change it here in that file to button color="warn" because looks like warn is orange, not red, please see difference: 
With that custom style: 
<img width="1725" alt="Screenshot 2024-07-15 at 10 13 44" src="https://github.com/user-attachments/assets/60bc7f59-aa0b-4d74-bb27-cfc1755b0575">

With color="warn":
<img width="1726" alt="Screenshot 2024-07-15 at 10 29 49" src="https://github.com/user-attachments/assets/e2138ff8-edf3-4935-8114-02d0b9a4b134">